### ISSUE TITLE
Episode FanArt Fallback not showing on AuraMod

### ISSUE THREAD

**Issue by Nuklear92** [SerpentDrago/skin.auramod#277]
So, when using Arctic Horizon, or any other skin whenever there's an Episode that doesn't have an official Artwork from TMDB and you have your FanArt service and key enabled within TMDb Helper or any other streaming addon you always get a fallback thumbnail from FanArt temporary instead of the TMDb's Background TV Show's Landscape imagine. 

In the case of Arctic Horizon 1-2 it will grab the FanArt thumb like from this TV Show's recent released episode which still haven't got the thumbnail updated. 

![image](https://user-images.githubusercontent.com/76958541/211610146-648d1134-40d2-44a2-86de-1b50ccb5cb5c.png)

Now, when I'm using AuraMod instead of grabbing the FanArt episode thumbnail as a fallback from the TV Show until TMDb update it, it still shows the background landscape from TV Show itself as you'll see in the screenshot below;

![image](https://user-images.githubusercontent.com/76958541/211610795-7ba60f54-6710-48e2-b9e3-2169432eb118.png)

To be clear, all other FanArt is being grabbed and displayed correctly. Like the Season's FanArt. As you'll see below, both AuraMod and Arctic Horizon display them correctly as you'll see on the left landscape box;

![image](https://user-images.githubusercontent.com/76958541/211611128-cf6826d5-898d-43d0-aff0-15146f249ebc.png)

![image](https://user-images.githubusercontent.com/76958541/211611158-7d441aa1-2470-42ac-9d49-ccaf45b7b9a5.png)

The only thing that AuraMod is not showing is the fallback Episode FanArt when TV Show Episodes haven't been updated with original thumbs. And when they do and either skin does with TMDb Helper or Addon's Background Refresh/Trakt Syncs then those Fallback FanArt Episode Thumbs gets switched back to the official TMDB's Episode Artwork. 

Hopefully this can be an easy fix for a future update of the skin. 👍

**Comment by Nuklear92** [comment]
@jojobrogess Hey, here's the issue I opened. Hopefully you can understand it a bit better now. 👍

**Comment by jojobrogess** [comment]
Yeah this definitely helps me understand what's going on a lot better.

I don't think it's a bug, but rather just how it was written.
Unless there is a skin setting somewhere about this kind of stuff, but I am unaware off the top of my head. 

Knowing that Arctic Horizon and others have this implemented correctly will help me see how they are doing it and come up with a solution. More than likely since this works as is, there will have to be a new setting added for it in skin settings. Unless I am incorrect in my assertion that this is not a bug and just how it was written. A lot of people use this skin, I would like to make sure stuff like that can be individually changed. Sadly at this moment I do not the whole process involved with adding an additional setting variable like that. But I am pretty sure I can figure out the image thing since others have it implemented, and TMDBH fur sure has documentation on it. 

*************************************************************************************************************************************

Thank you for posting the issue! 

I am a little overwhelmed with some of the other things I'm trying to fix/work on right now, but hopefully I can figure this out.

**Comment by Nuklear92** [comment]
> Yeah this definitely helps me understand what's going on a lot better.
> 
> 
> 
> I don't think it's a bug, but rather just how it was written.
> 
> Unless there is a skin setting somewhere about this kind of stuff, but I am unaware off the top of my head. 
> 
> 
> 
> Knowing that Arctic Horizon and others have this implemented correctly will help me see how they are doing it and come up with a solution. More than likely since this works as is, there will have to be a new setting added for it in skin settings. Unless I am incorrect in my assertion that this is not a bug and just how it was written. A lot of people use this skin, I would like to make sure stuff like that can be individually changed. Sadly at this moment I do not the whole process involved with adding an additional setting variable like that. But I am pretty sure I can figure out the image thing since others have it implemented, and TMDBH fur sure has documentation on it. 
> 
> 
> 
> *************************************************************************************************************************************
> 
> 
> 
> Thank you for posting the issue! 
> 
> 
> 
> I am a little overwhelmed with some of the other things I'm trying to fix/work on right now, but hopefully I can figure this out. 

Awesome, glad you understood it better. Yeah, no rush just wanted to point this out as it's been like this for years and no one seemed to had opened an issue about it. 

I'm not sure if you're planning to give this skin some small maintenance here and there. But, that's great as it's a very nice skin and like you said lots of people still use it. Specially for the Netflix style implementation it has, which if it kept evolving and being refined over the years it could had made this skin even better as regardless of all the settings and stuff it has in it maintain it's speed and reliability. 👍

**Comment by adamosborne83** [comment]
> Yeah this definitely helps me understand what's going on a lot better.
> 
> I don't think it's a bug, but rather just how it was written. Unless there is a skin setting somewhere about this kind of stuff, but I am unaware off the top of my head.
> 
> Knowing that Arctic Horizon and others have this implemented correctly will help me see how they are doing it and come up with a solution. More than likely since this works as is, there will have to be a new setting added for it in skin settings. Unless I am incorrect in my assertion that this is not a bug and just how it was written. A lot of people use this skin, I would like to make sure stuff like that can be individually changed. Sadly at this moment I do not the whole process involved with adding an additional setting variable like that. But I am pretty sure I can figure out the image thing since others have it implemented, and TMDBH fur sure has documentation on it.
> 
> Thank you for posting the issue!
> 
> I am a little overwhelmed with some of the other things I'm trying to fix/work on right now, but hopefully I can figure this out.

Just hijacking this thread, I see you've got some branches of Auramod, I'm having issues with AH2, so coming back to my first love...... which branch would you recommend?

**Comment by jojobrogess** [comment]
@adamosborne83 
> Just hijacking this thread, I see you've got some branches of Auramod, I'm having issues with AH2, so coming back to my first love...... which branch would you recommend?

Sorry I didn't see this, I assume you're talking to me?

Well none of the branches I have are really for public use, at least not yet. 
But that is the plan.
So I would suggest using the main branch at this repo, unless you're okay with some problems since idk what I'm doing lol

If you can somewhat read skin code, I would suggest you go individually through all the branches and look at the code or the descriptions and try to see what was updated to see if it's something you want, but that would be a pain obviously..
Since like I said, idk what I'm doing and I fur sure probably broke a lot more than I know of lol
so that would be the safest option.

*******************************************************************************************************************************

This branch is the cleanest(basic fixes with extremely low probability of breakage):
https://github.com/jojobrogess/skin.auramod/commits/Matrix-withFixes
But it has no realistic fixes atm. I have yet to push in any pr's that do major changes.


This is the code I'm running currently(EXTREME fixes and changes, with some broken image problems): 
https://github.com/jojobrogess/skin.auramod/tree/BROKENBRANCH-PersisonalAuraMod

But I DO NOT suggest it, since it WILL break some images for some viewtypes and some settings placement(where they are located) have been moved. It does have MY updates from this [branch](https://github.com/SerpentDrago/skin.auramod/commits/jojobrogess), which are pretty much cosmetic; and broken record, more than likely not written correctly.  

But if you use A LOT of genre icons, those should work now lol 
That's what I'm currently working on, genre icons and intern getting all widget type settings and viewmodes to work correctly.

Because atm, only some widgets display genre icons correctly, only some widgets display text labels correctly, only some widgets display clearlogos correctly, not having a submenu(like one not existing not just disabled) pretty much breaks most homestyle/viewmodes, and widgetlive is pretty much broken if you don't use it for live tv.

*******************************************************************************************************************************

I AM pretty sure, updating the addon.xml to display these dependency versions 
```
    <requires>
        <import addon="xbmc.gui" version="5.15.0"/>
        <import addon="script.skin.helper.service" version="1.20.0"/>
        <import addon="script.skin.helper.widgets" version="1.20.0"/>
        <import addon="script.skinshortcuts" version="2.0.3"/>
        <import addon="script.image.resource.select" version="3.0.2"/>
        <import addon="plugin.video.themoviedb.helper" version="5.0.50"/>
        <import addon="plugin.program.autocompletion" version="1.0.1"/>
        <import addon="resource.images.studios.white" version="0.0.32"/>
        <import addon="resource.images.studios.coloured" version="0.0.23"/>
        <import addon="resource.images.moviegenreicons.transparent" version="0.0.6"/>
        <import addon="resource.images.moviegenreicons.horizon" version="0.0.1"/>
        <import addon="script.colorbox" version="2.0.8"/>
    </requires> 
```
would help with getting kodi to run more smoothly in v20, without having to manually update them.

*******************************************************************************************************************************

Hopefully eventually I can get a more unified updated version going, it's just a lot of work to do while learning how to do it lol
